lerna.json

```diff
@@ -2,5 +2,11 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.0.0"
+  "version": "independent",
+  "workspaces": [],
+  "npmClient": "yarn",
+  "useWorkspaces": true,
+  "command": {
+    "tsc": "tsc"
```

Comment:
You could look at the [TypeORM](https://github.com/typeorm/typeorm) repo to see how they deliver their TS source to NPM 🙂